moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -299,6 +300,28 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
       field: "updated_at",
       render: entry => new Date(entry.updated_at).toLocaleDateString("en-US"),
     },
+    {
+      title: "Signal IDs",
+      field: "project_feature",
+      render: entry => {
```

Comment:
Being able to supply a custom rendering function here is top-shelf. This is a great use of it.